README.md

```diff
@@ -59,6 +59,21 @@ fs.createReadStream('data.csv')
   .pipe(fs.createWriteStream('data.json'));
 ```
 
+or from a string like
+
+```javascript
+var csv2json = require('csv2json');
+const { Readable } = require("stream")
+const readable = Readable.from(['csv content'])
+let csvText = ''
+readable.pipe(csv2json()).on('data', (chunk) => {
+	csvText += chunk.toString()
+})
+.on('end', () => {
+	let objects = JSON.parse(csvText)
```

Comment:
This module is for converting a CSV stream to a JSON stream, if you want JS objects instead, please use [`csv-parser`](https://github.com/mafintosh/csv-parser) directly :slightly_smiling_face: